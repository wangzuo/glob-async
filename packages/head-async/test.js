const head = require('./');

test('count', async () => {
  expect(await head('./file.txt')).toMatchSnapshot();
  expect(await head('./file.txt', { count: 3 })).toMatchSnapshot();
});

test('bytes', async () => {
  expect(await head('./file.txt', { bytes: 18 })).toMatchSnapshot();
});
