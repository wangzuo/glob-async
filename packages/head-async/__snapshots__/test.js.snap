// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bytes 1`] = `
"1
2
3
4
5
6
7
8
9
"
`;

exports[`count 1`] = `
Array [
  "1",
  "2",
  "3",
  "4",
  "5",
  "6",
  "7",
  "8",
  "9",
  "10",
]
`;

exports[`count 2`] = `
Array [
  "1",
  "2",
  "3",
]
`;
